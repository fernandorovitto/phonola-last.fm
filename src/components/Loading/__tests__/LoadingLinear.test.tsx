import 'jest-styled-components';
import LoadingLinear from '../LoadingLinear';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Test for LoadingLinear Component', () => {
  test('Should render the component', () => {
    const tree = renderer.create(<LoadingLinear text={'Loading'} />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
