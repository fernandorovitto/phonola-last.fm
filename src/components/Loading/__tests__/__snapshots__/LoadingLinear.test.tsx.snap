// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for LoadingLinear Component Should render the component 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 18px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 20px 0;
}

.c2 {
  width: 80%;
}

<div
  className="c0"
>
  <p
    className="c1"
  >
    Loading
  </p>
  <div
    className="MuiLinearProgress-root MuiLinearProgress-colorPrimary c2 MuiLinearProgress-indeterminate"
    role="progressbar"
  >
    <div
      className="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate"
      style={Object {}}
    />
    <div
      className="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
      style={Object {}}
    />
  </div>
</div>
`;
